neuralpredictors/regularizers.py

```diff
@@ -291,3 +295,27 @@ def forward(self, x, avg=False):
         out = out * (1 - self.gaussian2d.expand(1, 1, k1, k2).to(x.device))
 
         return agg_fn(out.pow(2)) / agg_fn(x.view(oc * ic, 1, k1, k2).pow(2))
+
+
+class Laplace1d(nn.Module):
+    def __init__(self, padding):
+        super(Laplace1d, self).__init__()
```

Comment:
Same here.

Reply:
Yes, why wouldn't it be the correct class name? Or do you mean the arguments in super()?

Reply:
Yes, I was talking about the arguments to `super`.